build/ci.go

```diff
@@ -839,6 +839,8 @@ func doXCodeFramework(cmdline []string) {
 	env := build.Env()
 
 	// Build the iOS XCode framework
+	build.MustRun(goTool("get", "-d", "golang.org/x/mobile/cmd/gomobile"))
```

Comment:
do we really need "-d" option? and if we do, I guess we don't need the line number 844 (duplicated).

Reply:
Oh we definitely need 844, it's deliberate. -d tells go to just download and not compile anything. 844 will actually compile it without downloading.